.yarnrc.yml

```diff
@@ -5,3 +5,5 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: "@yarnpkg/plugin-interactive-tools"
+
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
```

Comment:
Is this change necessary for the fix?

Reply:
no, sorry only the changed imports of date-fns